### DISCUSSION TITLE
How can I execute original sql string in the sqplite_orm library.

### DISCUSSION THREAD

**Discussion by CodingPanda87** [fnc12/sqlite_orm#788] [Q&A] [answered]
How can I execute original sql string in the sqplite_orm library. If sqplite_orm can do and not use sqlite3 API,it will be more perfect.

Thank you very much !!!

**Comment by fnc12** [comment, marked as answer]
> How can I execute original sql string in the sqplite_orm library.

No way. If you want to execute raw string query please use libsqlite3 instead cause it is designed for that. `sqlite_orm` is designed to eliminate usage of raw string queries and replace it with nice DSL which guarantees that you will never make a typo

> If sqplite_orm can do and not use sqlite3 API,it will be more perfect

`sqlite_orm` uses sqlite3 API. There is no alternative SQLite3 engines in the world

**Reply by CodingPanda87** [reply]
I got it ,thanks!
